src/Lifecycle.ts

```diff
@@ -824,6 +825,8 @@ async function startMatrixClient(startSyncing = true): Promise<void> {
     // Now that we have a MatrixClientPeg, update the Jitsi info
     await Jitsi.getInstance().start();
 
+    AutoRageshakeStore.instance.start();
```

Comment:
Given that changing the setting forces a page reload, it'd be nicer if we only start it if the setting allows for it, then the store itself can run unconditionally